solidity/src/BftBridge.sol

```diff
@@ -126,6 +126,9 @@ contract BFTBridge {
     // Mapping from Base tokens to Wrapped tokens
     mapping(address => bytes32) private _baseTokenRegistry;
```

Comment:
I think it's enough to make :
  
 mapping(bytes32 => address) private _erc20TokenRegistry;
 mapping(address => bytes32) private _baseTokenRegistry;
    
 both public. This should also handle the accessories